frontend/src/pages/ForgetPassword/ForgetPassword.jsx

```diff
@@ -0,0 +1,134 @@
+import { useState, useEffect, useRef } from 'react';
+import { Container, Form, Button } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+
+import styles from './ForgetPassword.module.css';
+
+import { forgetPassword } from '../../api';
+import FatherSonBlock from '../../components/FatherSonBlock';
+import FormEmailInput from '../../components/form/FormEmailInput';
+import Header from '../../components/Header/Header';
+import MessageBar from '../../components/MessageBar';
+import TextLink from '../../components/TextLink';
+
+import logoEmailSent from '../../media/icons/vector.png';
```

Comment:
I think we have duplicated the envelope and checkMark icons in /media/icons and img/. Probably its better idea to keep them in media/icons folders. 
After we merge this branch we can clean up and remove duplicated icons